PendingReleaseNotes

```diff
@@ -306,4 +306,9 @@
 
 * RGW S3: Support has been added for BlockPublicAccess set of APIs at a bucket
   level, currently blocking/ignoring public acls & policies are supported.
-  User/Account level apis are planned to be added in the future
\ No newline at end of file
+  User/Account level apis are planned to be added in the future
```

Comment:
[off-topic] we need to fix the document to use APIs instead of apis to be consistent in another PR, as ReleaseNotes is use facing document.

Reply:
on it!

Reply:
why? i think **Pending**ReleaseNotes are note released yet. so no need to backport them, i guess.

Reply:
yeah, thanks for clarifying :+1: